packages/svelte/tests/migrate/samples/slots-custom-element/input.svelte

```diff
@@ -0,0 +1,23 @@
+<svelte:options customElement="my-element" />
+
+<script>
+	// to show that it doesn't bail out from the whole migration
+	let count = 0;
+</script>
+
+<button on:click={()=>count++}><slot /></button>
+
+{count}
+
+{#if foo}
+	<slot name="foo" {foo} />
+{/if}
+
+{#if $$slots.bar}
+	{$$slots}
```

Comment:
This begs an interesting question ... what about `$$slots` in the context of custom elements? `$$slots` is also legacy/deprecated syntax and as such shouldn't be used anymore. Is it even feasible/necessary to use that in the context of custom elements? @Theo-Steiner any real world use cases from your end?

Reply:
I had exactly the same question but when i tried to use `$$slots` in a custom element it's actually populated so it might be used? I don't think anyone uses it but i would still leave it untouched in case of custom elements because the migration removes it in favor of checking the prop for falsy values which will not be here if we don't migrate slots.

Reply:
It's a bit unfortunate because it may mean that after `$$slots` are gone people will have to migrate away from it, using something like https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement/assignedNodes instead. As you say it's probably super rare in practise, but it's something to keep in mind.

Reply:
this just means
`$$slots.bar`
will become
`$host().shadowRoot.querySelectorAll("slot[name=bar]")`
which is not too bad either IMO